ksql-cli/src/test/java/io/confluent/ksql/CliTest.java

```diff
@@ -445,4 +451,22 @@ public void shouldRegisterRemoteCommand() {
     assertThat(terminal.getCliSpecificCommands().get("server"),
         instanceOf(Cli.RemoteServerSpecificCommand.class));
   }
+
+  @Test
+  public void shouldPrintErrorOnUnsupportedAPI() {
+    KsqlRestClient mockRestClient = EasyMock.mock(KsqlRestClient.class);
+    EasyMock.expect(mockRestClient.makeRootRequest()).andReturn(
+        RestResponse.erroneous(
+            new KsqlErrorMessage(
+                NOT_ACCEPTABLE.getStatusCode() * Errors.HTTP_TO_ERROR_CODE_MULTIPLIER,
```

Comment:
again, consider making this multiplication a function call: `Errors.toErrorCode(NOT_ACCEPTABLE)`